public/data/nazmulhassan.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "MD NAZMUL HASSAN",

```

Comment:
```suggestion
    "name": "MD Nazmul Hassan",
```

It looks better when its small, it shouldn't be all caps.

Reply:
What does MD stand for?

Reply:
> What does MD stand for?

Muhammad

Reply:
Then, I suggest it me Muhammad Nazmul Hassan instead of MD

```suggestion
    "name": "Nazmul Hassan Nazmul Hassan",

```

Reply:
> Then, I suggest it me Muhammad Nazmul Hassan instead of MD

Please commit this @nazmul19

Reply:
> Then, I suggest it me Muhammad Nazmul Hassan instead of MD

It's their profile, they can use the name they choose

Reply:
Yes true, deleting the suggestion.